Asap2/Asap2.csproj

```diff
@@ -64,70 +35,33 @@
     <Asap2Parser>$(ProjectDir)Asap2</Asap2Parser>
   </PropertyGroup>
   <ItemGroup>
-    <None Include="..\README.md" Pack="true" PackagePath="\docs"/>
+    <None Include="..\README.md" Pack="true" PackagePath="\docs" />
     <PackageReference Include="NuGet.Build.Tasks.Pack" Version="6.1">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="YaccLexTools" Version="0.2.2">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Xml" />
   </ItemGroup>
+
+
+  <PropertyGroup>
+    <AnalysisMode>AllEnabledByDefault</AnalysisMode>
+  </PropertyGroup>
+
   <ItemGroup>
-    <Compile Include="Asap2Tree\Asap2Base.cs" />
```

Comment:
Why are all project files not included anymore?

Reply:
I had to change the project file format to the new SDK format. With this format, all files (including subfolders) are included by convention by default.
More info here: https://learn.microsoft.com/en-us/dotnet/core/project-sdk/overview